### PR TITLE
refact rgw pools creation on containerized deployment

### PR DESCRIPTION
**Pull request by guits**
rgw: refact rgw pools creation

Refact of 8704144
There is no need to have duplicated tasks for this. The rgw pools
creation should be delegated on a monitor node se we don't have to care
if the admin keyring is present on rgw node.
By the way, only one task is needed to create the pools, we just need to
use the `docker_exec_cmd` fact already defined in `ceph-defaults` to
achieve it.

Closes: https://bugzilla.redhat.com/show_bug.cgi?id=1550281

### CODE REVIEW

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-docker_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-centos7_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-centos7_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-docker_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-ooo_collocation

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-docker_cluster_collocation

**Comment by leseb** [comment]
don't we have this? https://github.com/ceph/ceph-ansible/pull/2424

**Comment by guits** [comment]
@leseb this is a refact of #2424 because part of our comments hasn't been addressed properly.

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-centos7_cluster

File: roles/ceph-rgw/tasks/main.yml [hunk 1]
```diff
@@ -51,3 +38,13 @@
     - containerized_deployment
   # Hard code this so we will skip the entire file instead of individual tasks (Default isn't Consistent)
   static: False
+
+- name: create rgw pools if rgw_create_pools is defined
+  command: "{{ docker_exec_cmd }} ceph --connect-timeout 5 --cluster {{ cluster }} osd pool create {{ item.key }} {{ item.value.pg_num }}"
+  changed_when: false
+  failed_when: false
```
Comment (leseb) [inline roles/ceph-rgw/tasks/main.yml]: why?

Comment (guits) [reply roles/ceph-rgw/tasks/main.yml]: leftover from merge conflict

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-centos7_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-docker_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-docker_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-centos7_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-docker_cluster_collocation

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-ooo_collocation

**Pull request merged**